config/locales/application_form.yml

```diff
@@ -221,3 +221,15 @@ en:
             start_date:
               invalid: Enter a start date in the correct format, for example 5 2018
               before: Enter a start date that is before the end date
+        candidate_interface/gcse_qualification_details_form:
+          attributes:
+            grade:
+              blank: Enter the grade
+            award_year:
+              blank: Enter the year
+        candidate_interface/gcse_qualification_type_form:
+          attributes:
+            qualification_type:
+              blank: Enter the type of qualification
+            level:
+              blank: Enter your level
```

Comment:
Is this necessary? Level is something we use internally and is not candidate facing.